lib/main.dart

```diff
@@ -1,7 +1,30 @@
-import 'package:catcher/catcher_plugin.dart';
-import 'package:sirene/services/logging.dart';
+import 'package:flutter/widgets.dart';
 
-import './app.dart';
+import 'package:firebase_crashlytics/firebase_crashlytics.dart';
 
-final config = CatcherOptions(SilentReportMode(), [LoggingCatcherHandler()]);
-void main() => Catcher(AppWidget(), debugConfig: config, releaseConfig: config);
+import 'package:sirene/app.dart';
+import 'package:sirene/interfaces.dart';
+
+void main() {
+  FlutterError.onError = (details) {
+    var isDebug = false;
+    assert(isDebug = true);
+
+    if (isDebug) {
+      FlutterError.dumpErrorToConsole(details);
+    } else {
+      final user = App.locator != null
+          ? App.locator.get<LoginManager>().currentUser
+          : null;
```

Comment:
I think this could be expressed simply by
```dart
final user = App.locator?.get<LoginManager>().currentUser;
```

Reply:
@apaatsio Ah cool, thanks for the tip!